intranet/apps/announcements/notifications.py

```diff
@@ -113,9 +115,13 @@ def announcement_posted_email(request, obj, send_all=False):
     if settings.EMAIL_ANNOUNCEMENTS:
         subject = "Announcement: {}".format(obj.title)
         if send_all:
-            users = get_user_model().objects.all()
+            users = get_user_model().objects.exclude(Q(user_type="service") | Q(graduation_year__gte=get_senior_graduation_year()))
         else:
-            users = get_user_model().objects.filter(receive_news_emails=True)
+            users = (
+                get_user_model()
+                .objects.exclude(Q(user_type="service") | Q(graduation_year__gte=get_senior_graduation_year()))
```

Comment:
Same here - do we want to include only students, teachers, and counselors; instead of excluding service users; and this would exclude current seniors

You _could_ use `graduation_year__gt=get_senior_graduation_year()` if you wanted to use `.exclude()`